src/snagflash/dfu.py

```diff
@@ -34,7 +34,7 @@ def dfu_cli(args):
 	dev.default_timeout = int(args.timeout)
 	for dfu_config in args.dfu_config:
 		(altsetting,sep,path) = dfu_config.partition(":")
-		if args.size:
+		if hasattr(args, "size"):
```

Comment:
This entire if/else block and the `if size is None` check a few lines below will be unecessary. Could you please remove those?

Reply:
If you could also remove the relevant doc section (lines 37 to 41 in doc/snagflash.md) that would also be great.

Reply:
removed from docs as well